sympy/core/function.py

```diff
@@ -1641,7 +1641,11 @@ def derivative_count(self):
 
     @property
     def free_symbols(self):
```

Comment:
I think this looks right. I was wondering if `Expr.expr_free_symbols` was something related but I don't think so.

Reply:
`Derivative(x, y, evaluate=False).free_symbols` will only return `{x}`, but I think that is probably OK (same as before).